src/Hive.Core/sia380/sia380.py

```diff
@@ -388,41 +429,63 @@ def calculate_step(t,
         Q_s_tr = sum(Q_s_tr_per_surface[t])
         Q_s = Q_s_tr   # currently, only transparent surfaces
         
+        if t == 4074:
+            print()
+        
         if eta_g_t is None:
             # Heatgains/-losses ratio (Wärmeeintrag/-verlust Verhältnis), gamma
             # calculating for different cases, heating / cooling, upper / lower bounds, with / without heat recovery
             gamma_ub = calc_gamma(Q_i, Q_s, Q_T_ub, Q_V_ub)
             gamma_lb = calc_gamma(Q_i, Q_s, Q_T_lb, Q_V_lb)
             gamma_ub_no_hr = calc_gamma(Q_i, Q_s, Q_T_ub, Q_V_ub_no_hr)
             gamma_lb_no_hr = calc_gamma(Q_i, Q_s, Q_T_lb, Q_V_lb_no_hr)
+            gamma_ub_nat_vent = calc_gamma(Q_i, Q_s, Q_T_ub, Q_V_ub_nat_vent)
+            gamma_lb_nat_vent = calc_gamma(Q_i, Q_s, Q_T_lb, Q_V_lb_nat_vent)
```

Comment:
maybe not anymore for this PR, but in some refactoring... we keep adding more and more `gamma` cases. can we do this more elegantly?

Reply:
yeah it is getting unwieldy but i am not sure how to avoid it at the moment. let's see if an idea comes when refactoring